themes/custom/ts_wrin/sass/components/_entity-embed.scss

```diff
@@ -0,0 +1,59 @@
+.embedded-entity {
+  border-radius: 6px;
+  padding: var(--spacer-1) var(--spacer-6) var(--spacer-4);
+  @include mq($md) {
+    padding: var(--spacer-2) var(--spacer-15) var(--spacer-5);
+  }
+
+  @include font($acumin-semi-cond, "light");
+  @include font-line-height(16, 21);
+
+  &.has-blue-background-color,
+  &.has-green-background-color,
+  &.has-teal-background-color,
+  &.has-black-background-color {
+    color: $white;
+  }
+  form.webform-submission-form {
+    input[type="text"],
+    input[type="email"] {
+      @include font($acumin-semi-cond, "regular");
+      @include font-line-height(16, 18);
+      height: 46px;
+      padding: 15px 10px 12px;
+      background-color: $white;
+      width: 100%;
+    }
+  }
+  .form-type-processed-text {
+    @include font($acumin-semi-cond, "regular");
+    @include font-line-height(16, 18);
+    @include mq($md) {
+      @include font-line-height(22, 28);
+    }
+  }
+  h4, h5, p {
+    font-family: inherit;
+    font-size: inherit;
+    margin: 16px 0;
+  }
+  .field--name-field-other-orto-list-tags {
+    font-size: 0;
+    visibility: hidden;
+  }
+}
+
+.article .embedded-entity .form-type-processed-text {
+  @include font($caslon, "light");
+  @include font-line-height(16, 18);
+  @include mq($md) {
+    @include font-line-height(22, 28);
+  }
+}
+
+.embed--block-type--in-this-series {
+  padding: 0;
```

Comment:
Let's leave the color of `.teaser-title a` as black within this element. It's only getting overridden because it's embedded within the `.bullet` logic. If you change the rule about the `.bullet` links to only apply within a `li` it should be good.